src/rtdev.erl

```diff
@@ -764,3 +764,20 @@ get_node_logs() ->
           {ok, Port} = file:open(Filename, [read, binary]),
           {lists:nthtail(RootLen, Filename), Port}
       end || Filename <- filelib:wildcard(Root ++ "/*/dev/dev*/log/*") ].
+
+-ifdef(TEST).
+
+release_versions_test() ->
+    ok = rt_config:set(rtdev_path, [{root, "/Users/hazen/dev/rt/riak"},
```

Comment:
If you use `os:getenv("RT_DEST_DIR")` in place of hardcoded path, all Linux users will say thank you (as well as all those whose login is not 'hazen':)